.pre-commit-config.yaml

```diff
@@ -26,6 +26,11 @@ repos:
         language: script
         types: [python]
         require_serial: true
+  - repo: https://github.com/asottile/pyupgrade
```

Comment:
Thanks, i wasn't aware of this tool.

FYI I manage this file upstream via `cruft` since its the same across a bunch of repos I have so I will consider adding it there too https://github.com/allenporter/cookiecutter-python